src/scripts/main.js

```diff
@@ -1 +1,51 @@
 'use strict';
+
+const clicks = ['click', 'contextmenu'];
```

Comment:
The `clicks` array includes 'contextmenu', which corresponds to a right-click event. Ensure that the `secondPromise` only resolves on left or right clicks as specified in the task description.